resharper/resharper-unity/src/Yaml/Psi/UnityYamlLanguage.cs

```diff
@@ -5,12 +5,12 @@
 namespace JetBrains.ReSharper.Plugins.Unity.Yaml.Psi
 {
     [LanguageDefinition(Name)]
-    public class UnityYamlLanguage : YamlLanguage
+    public class UnityYamlLanguage : KnownLanguage
```

Comment:
I'm now confused with the relationship between `YamlLangauge` and `UnityYamlLanguage`. Which file types are used for what files? Might be worth commenting